### PR TITLE
Fix rounding down of the RN50 PyTorch example epoch length

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
 - fixes rounding down of the RN50 PyTorch example epoch length by using math.ceil()

#### What happened in this PR?
 - fixes rounding down of the RN50 PyTorch example epoch length by using math.ceil()
 - ad https://github.com/NVIDIA/DALI/issues/1427
 - this affects only warmup learning rate adjustment in RN50 training script so the difference should be significant (if any in the most cases)
 - tested in CI

**JIRA TASK**: [NA]

### CODE REVIEW

**Comment by JanuszL** [comment]
!build
